packages/strapi-generate-new/files/.npmrc

```diff
@@ -1 +0,0 @@
-package-lock=false
```

Comment:
now we need the new projects to have their lock files wether it's yarn or npm

Reply:
Is it Strapi or Heroku that automatically creates a yarn.lock file if yarn is installed on the OS? Because this is an issue I had to deal with unexpectedly.

Reply:
Strapi generates a yarn.lock file. It is the itended behavior :) If using npm it will use a package-lock.json file